Documentation/using-dex.md

```diff
@@ -137,18 +137,16 @@ Matcher symbols will only function if all of the following conditions are met:
 
 1. application is confidential (i.e. not public)
 
-2. URL scheme is `http` or `https`
-
-3. A TLD domain, and a first subdomain must be present in URL, and neither of these two domain name sections can include a matcher symbol. 
+2. A TLD domain, and a first subdomain (SLD) must be present in URL, and neither of these two domain name sections can include a matcher symbol. 
 
 * Invalid: `https://*hello.com`: matcher symbol is contained in the first subdomain below the TLD 
 * Valid: `https://*.engineer.learning.com`
 * Valid: `https://*.learning.com`
 * Valid: `https://**.learning.com`
-* Valid: `https://abc**.learning.com` : will match abc.def.ghf.learning.com 
+* Valid: `https://abc**.learning.com` : will match abc.d.learning.com abcd.learning.com
```

Comment:
Define behaviour on edges when both scenarios make sense: * should be at least on character